CHANGELOG.rst

```diff
@@ -4,6 +4,20 @@ Ansible Playbooks for Confluent Platform - Release Notes
 
 .. contents:: Topics
 
+v7.8.2
+======
+
+Notable enhancements
+-------------
+- Removed kafka kraft delegation for mTLS clusters
```

Comment:
I think we should explain it a bit. Something like removed kraft controller and broker dependency on each other for RBAC enabled clusters with mTLS